.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: ['master']
+    tags:
+      - '*'
```

Comment:
```suggestion

```
The actions seem to not be running

Reply:
It doesn't work because we wouldn't contact to INFRA

Reply:
> It doesn't work because we wouldn't contact to INFRA

I think I can create an action in repo